app/examples/qutebrowser

```diff
@@ -0,0 +1,111 @@
+" Options
+set adblocker=update
+set devtoolsposition=split
+set downloadmethod=confirm
+set firefoxmode=always
+set fontsize=16
+set norestoretabs
+set permissioncamera=ask
+set permissiondisplaycapture=ask
+set permissionmediadevices=allow
+set permissionmicrophone=ask
+set permissionpersistentstorage=allow
+set redirecttohttp
+set tabclosefocusright
+set tabreopenposition=next
+set windowtitle=full
+set mapsuggestposition=bottomleft
+
+" Mappings
+nmap <Up> <action.scrollUp>
+nmap <Down> <action.scrollDown>
+nmap <Left> <action.scrollLeft>
+nmap <Right> <action.scrollRight>
+nmap <PageUp> <action.scrollPageUp>
+nmap <PageDown> <action.scrollPageDown>
+nmap <Home> <action.scrollTop>
+nmap <End> <action.scrollBottom>
+
+nmap o <action.toExploreMode>
+nmap wi <:devtools>
+nmap <C-PageUp> <action.previousTab>
+nmap <C-PageDown> <action.nextTab>
+nmap <F5> <action.reload>
+nmap <C-F5> <action.reloadWithoutCache>
+nmap Sh <:history>
+nmap Ss <:help>
+nmap <C-A-p> <:print>
+nmap <C-q> <:quit>
+nmap <C-W> <:quit>
+nmap ZQ <:quit>
+nmap gd <:write>
+nmap <C-Tab> <action.nextTab>
+nmap <C-S-PageUp> <action.moveTabBackward>
+nmap <C-S-PageDown> <action.moveTabForward>
+nmap <C-S-Tab> <action.previousTab>
+nmap <C-I> <:devtools>
+nmap <C-T> <action.reopenTab>
+nmap <A-1> <:b 0>
+nmap g0 <:b 0>
+nmap g^ <:b 0>
+nmap g$ <:b 9999>
+nmap <A-2> <:b 1>
+nmap <A-3> <:b 2>
+nmap <A-4> <:b 3>
+nmap <A-5> <:b 4>
+nmap <A-6> <:b 5>
+nmap <A-7> <:b 6>
+nmap <A-8> <:b 7>
+nmap <A-9> <:b 9999>
+nmap <C-^> <action.toLastUsedTab>
+nmap <C-Tab> <action.toLastUsedTab>
+nmap ga <action.openNewTab>
+nmap gC <action.openNewTabWithCurrentUrl>
+nmap gJ <action.moveTabForward>
+nmap gK <action.moveTabBackward>
+nmap <C-w> <:close>
```

Comment:
This will prevent all window split mappings from being accessed, as those are prefixed with `<C-w>`, we might want to reconsider including this even if it's possible in qutebrowser.

Reply:
Hmm.. Maybe I could comment it out?

Reply:
That would be alright I think, just make sure to also add a comment above as to why it's disabled, you can refer the user to the following section for a detailed explanation:  `:h spitting`

Reply:
I've added the comments. Feel free to suggest changes to the wording.